ws-nextjs-app/utilities/pageRequests/getPageData.ts

```diff
@@ -0,0 +1,84 @@
+import { BFF_FETCH_ERROR } from '#app/lib/logger.const';
+import getToggles from '#app/lib/utilities/getToggles';
+import { FetchError } from '#app/models/types/fetch';
+import PageDataParams from '#app/models/types/pageDataParams';
+import constructPageFetchUrl, {
+  UrlConstructParams,
+} from '#app/routes/utils/constructPageFetchUrl';
+import sendCustomMetric from '#server/utilities/customMetrics';
+import { NON_200_RESPONSE } from '#server/utilities/customMetrics/metrics.const';
+import { PageTypes } from '#app/models/types/global';
+import fetchPageData from '#app/routes/utils/fetchPageData';
+import getAgent from '../undiciAgent';
+
+type LoggerType = {
+  error: (id: string, params: { [key: string]: string | number }) => void;
+};
+
+const getPageData = async (
+  { id, service, rendererEnv, resolvedUrl }: PageDataParams,
+  constructUrlParams: UrlConstructParams,
+  logger: LoggerType,
+  pageType: PageTypes,
+) => {
+  const pathname = `${id}${rendererEnv ? `?renderer_env=${rendererEnv}` : ''}`;
+  const livePageUrl = constructPageFetchUrl(constructUrlParams);
+
+  // UNCOMMENT ONCE A TEST ASSET IS ON BFF PRODUCTION
+  // import certsRequired from '#app/routes/utils/certsRequired';
+  // import getEnvironment from '#app/routes/utils/getEnvironment';
+  //
+  // const env = getEnvironment(pathname);
+  // const optHeaders = { 'ctx-service-env': env };
+  // const agent = certsRequired(pathname) ? await getAgent() : null;
+
+  // FOR NOW USE THIS:
+  // MAKE SURE BFF_PATH IS SET TO https://fabl.api.bbci.co.uk/preview/module/simorgh-bff
+  const optHeaders = { 'ctx-service-env': 'live' };
```

Comment:
I think we can remove this as the default is always live, unless we need to override it (which is then determined by the value of the `renderer_env`